glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
@liuy see comment above yours for why it exists. The netlink interface cannot return failures. If open fails you end up with no device in runner, but you still get a device in the kernel. That then can get reported to initiators in the REPORT LUNS info.

So it seems like a hack. You run check_config first via a dbus call. If it fails, then you know not to call open and you avoid the problem.

Above, I was saying to just handle the problem at open time, we need to fix the netlink interface.